var/spack/repos/builtin/packages/pixman/package.py

```diff
@@ -25,6 +25,8 @@ class Pixman(AutotoolsPackage):
     version("0.32.6", sha256="3dfed13b8060eadabf0a4945c7045b7793cc7e3e910e748a8bb0f0dc3e794904")
 
     depends_on("pkgconfig", type="build")
+    depends_on("flex", type="build")
+    depends_on("bison@3:", type="build")
```

Comment:
Do these dependencies apply to all of the versions or only for those from some release on?

Reply:
As far as I know, all of them? It's an iteration in the right direction; if someone discovers an incompatibility with an older version we can improve the constraints then I think